### ISSUE TITLE
Ability to connect to an existing docker container from "@docker" in CLI

### ISSUE THREAD

**Issue by amagee** [Fizzadar/pyinfra#455]
If I'm playing around with a container ID that changes as I go, currently it seems I have to put the container ID in an inventory file, since the `@docker/...` syntax in the CLI assumes the thing that follows the slash is an image. So I have to do something like this:

```
# inventory.py
servers = [
    ('@docker/ubuntu', {'docker_container_id': "my-container-id"}),
]
```

and run `pyinfra inventory.py deploy.py` to run against my docker container `my-container-id`.

It would be nice to be able to run `pyinfra @docker/my-container-id deploy.py` (or something like that; I guess there should be some way to disambiguate container IDs from image IDs).

This isn't really a big deal, though it was a bit difficult for me to figure out since when I decided I wanted to run against a container ID, I hadn't done anything with an `inventory.py` file (I'd just used `@local` and `@docker/ubuntu`) and the documentation of `@docker` [here](https://docs.pyinfra.com/en/latest/connectors.html#docker) doesn't help much in finding out where the `docker_container_id` goes.

**Comment by Fizzadar** [comment]
@amagee I'm keen to provide support for doing this via the CLI directly, seems very sensible. I believe it should be possible to accept both types simply by running `docker inspect <ID>` and identifying if the object is an image or a running container.

**Comment by Fizzadar** [comment]
Also - a possible and immediate workaround would be to use an environment variable, ie `os.envar['DOCKER_CONTAINER_ID']` in the inventory and pass that via the CLI. Should work until I get this implemented!

**Comment by blarghmatey** [comment]
Another useful addition to this connector would be the ability to specify any labels/tags to be applied to the generated image. Whether via the inventory or CLI. Ideally I could write an inventory file that specifies a base image and then populate the desired labels to be applied to it as part of the host data? This could then serve as an alternative means of identifying the container that I want to keep operating on, vs having to manually populate the ID after the first run from the CLI.

**Comment by Fizzadar** [comment]
Specifying container ID has been implemented in https://github.com/Fizzadar/pyinfra/commit/0fe3acf27f984008dbe0a3a833e81a3a68e79520 and released in v1.3!
